stdlib/3/tempfile.pyi

```diff
@@ -25,7 +31,7 @@ def TemporaryFile(
     newline: Optional[str] = ...,
     suffix: Optional[AnyStr] = ...,
     prefix: Optional[AnyStr] = ...,
-    dir: Optional[AnyStr] = ...,
+    dir: Optional[_PathType] = ...,
```

Comment:
You must use `_PathType[AnyStr]` here and elsewhere.

Reply:
Why is that? _PathType doesn't have parameters, it's just a union.